docs/ref/forms/fields.txt

```diff
@@ -395,7 +395,7 @@ be ignored in favor of the value from the form's initial data.
 .. versionadded:: 5.0
 
 The ``template_name`` argument allows a custom template to be used when the
-field is rendered with :meth:`~django.forms.BoundField.as_field_group`. By 
+field is rendered with :meth:`~django.forms.BoundField.as_field_group`. By
```

Comment:
I don't know how to commit a change to this file without adding this whitespace change.